tests/PhpUnits/Helpers/MetaSeoTest.php

```diff
@@ -58,15 +57,15 @@ public function testRenderFullSuccess() {
 				"PHP",
 			]);
 			MetaSeo::SetDescription("Mario Rawady is a Software Engineer");
-			MetaSeo::SetPhoto("https://rawadymario.com/assets/img/logo-big.png");
-			MetaSeo::SetUrl("https://rawadymario.com");
+			MetaSeo::SetPhoto("https://dg-splash.com/assets/img/logo-big.png");
+			MetaSeo::SetUrl("https://dg-splash.com");
 			MetaSeo::SetRobots(true);
 			MetaSeo::SetGoolgeSiteVerification("");
 			MetaSeo::SetCopyright("2022. Mario Rawady");
```

Comment:
Let's change here "Mario Rawady" to "Digital Splash". Make sure to change it in the Expected File too (We will check this together)